src/niveristand/clientapi/realtimesequence.py

```diff
@@ -123,10 +123,11 @@ def _transform(self):
         rtsequtils.compile_rtseq(self._rtseq)
 
     def _update_references(self):
-        self._ref = Reference(str(self), self._build_file_name())
+        self._ref.append(Reference(str(self), self._build_file_name()))
         self._rtseq.References = References()
         for referenced in self._rtseqpkg.get_referenced(self):
             self._rtseq.References.AddReference(referenced.get_reference())
```

Comment:
Why not using pop here instead of changing get_reference?

Reply:
So I should leave get_reference as it was and just call pop on the returned value?